### DISCUSSION TITLE
the surround-nvim has a "Cycle surrounding quotes type stq" how to achive that?

### DISCUSSION THREAD

**Discussion by voyeg3r** [kylechui/nvim-surround#79] [Ideas]
The `stq` has a mnemonic like: "surround toggle quote", would it be easy to achieve?

**Comment by kylechui** [comment]
This is a pretty good idea, although perhaps to better fit with vim-surround mnemonics it might be better to switch it to `ytq` for "you toggle quote"? In any case, see #58 as I need to redo the user interface to accommodate for all the ways people want to use this plugin. Thanks!

**Reply by kylechui** [reply]
@voyeg3r This is now under consideration via #98.
